frontend/src/components/Header/components/HeaderLogo.tsx

```diff
@@ -15,7 +15,7 @@ const HeaderLogo = ({ type }: LogoButtonProps) => {
   return (
     <>
       <button type="button" className="flex flex-row items-center gap-4" onClick={moveToMainPage}>
-        <img src={logoSmall} />
+        <img src={logoSmall} alt="로고" />
```

Comment:
좀 더 명확하게 "Boarlog 로고"는 어떤가요?
```suggestion
        <img src={logoSmall} alt="Boarlog 로고" />
```

Reply:
감사합니다! Main 작업하면서 수정해보겠습니다 :)

Reply:
의견 반영해주셔서 감사합니다!